cockatrice/src/player.cpp

```diff
@@ -1972,6 +1976,49 @@ bool Player::clearCardsToDelete()
     return true;
 }
 
+void Player::actMoveCardXCardsFromTop()
+{
+    bool ok;
+    int number = QInputDialog::getInt(0, tr("Place card X cards from top library"), tr("How many cards from the top of the deck should this card be placed:"), defaultNumberTopCardsToPlaceBelow, 1, 2000000000, 1, &ok);
```

Comment:
Is this X cards ABOVE our card or X-1?  Unexpectedly Absent says "just beneath the top X cards of your library" whereas Oust says `second from the top` which is clarified as `6/15/2010: If the targeted creature’s owner has one or more cards in his or her library, that creature is put into that library directly under the top card.`